examples/qml_features/rust/src/invokables.rs

```diff
@@ -13,23 +13,38 @@ pub mod ffi {
         include!("cxx-qt-lib/qcolor.h");
         /// QColor from cxx_qt_lib
         type QColor = cxx_qt_lib::QColor;
+
+        include!(<QObject>);
+        /// QObject from Qt
+        type QObject;
     }
+    impl UniquePtr<QObject> {}
 
     /// A QObject which has Q_INVOKABLEs
-    #[cxx_qt::qobject(qml_uri = "com.kdab.cxx_qt.demo", qml_version = "1.0")]
+    #[cxx_qt::qobject]
     pub struct RustInvokables {
         red: f32,
         green: f32,
         blue: f32,
     }
 
-    impl Default for RustInvokables {
-        fn default() -> Self {
-            Self {
-                red: 0.0,
-                green: 0.4667,
-                blue: 0.7843,
-            }
+    impl cxx_qt::Constructor<(f32, f32, f32, UniquePtr<QObject>)> for qobject::RustInvokables {
+        type BaseArguments = (*mut QObject,);
```

Comment:
This looks good enough for me to port https://github.com/KDAB/cxx-qt/blob/d50012461d76d69884be82941f3b36ae915dea14/examples/qml_features/rust/src/custom_parent_class.rs over to show  how to do a custom base argument so that `QQuickPaintedItem` is possible.

I'll do this once things have been rebased here.